### ISSUE TITLE
WriteNewLocaleFileOnStart: true

### ISSUE THREAD

**Issue by C4BR3R4** [Rsl1122/Plan-PlayerAnalytics#281]
Instructions for issue reporting:
- When the "WriteNewLocaleFileOnStart: true" value is triggered to generate a language file, the plugin fails to reload or start.
- https://hastebin.com/jogifonavu

**Comment by Fuzzlemann** [comment]
Cannot reproduce with PaperSpigot 1.12.
Can you send any stacktrace that is possibly thrown on reload / start?

**Comment by Rsl1122** [comment]
Reproduced issue on Spigot 1.11.2

```
Error occurred while enabling Plan v3.6.4 (Is it up to date?)
java.lang.NoSuchMethodError: org.apache.commons.lang3.StringUtils.compare(Ljava/lang/String;Ljava/lang/String;)I
        at main.java.com.djrapitops.plan.utilities.comparators.LocaleEntryComparator.compare(LocaleEntryComparator.java:20) ~[?:?]
        at main.java.com.djrapitops.plan.utilities.comparators.LocaleEntryComparator.compare(LocaleEntryComparator.java:16) ~[?:?]
        at java.util.TimSort.countRunAndMakeAscending(Unknown Source) ~[?:1.8.0_131]
        at java.util.TimSort.sort(Unknown Source) ~[?:1.8.0_131]
        at java.util.Arrays.sort(Unknown Source) ~[?:1.8.0_131]
        at java.util.stream.SortedOps$SizedRefSortingSink.end(Unknown Source) ~[?:1.8.0_131]
        at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:1.8.0_131]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:1.8.0_131]
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) ~[?:1.8.0_131]
        at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:1.8.0_131]
        at java.util.stream.ReferencePipeline.collect(Unknown Source) ~[?:1.8.0_131]
        at main.java.com.djrapitops.plan.locale.Locale.writeNewDefaultLocale(Locale.java:99) ~[?:?]
        at main.java.com.djrapitops.plan.locale.Locale.loadLocale(Locale.java:69) ~[?:?]
        at main.java.com.djrapitops.plan.Plan.onEnable(Plan.java:128) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:271) ~[spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:376) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:326) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:421) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:382) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:337) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:272) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:544) [spigot-1.11.2.jar:git-Spigot-3fb9445-6e3cec8]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_131]
```

**Comment by Rsl1122** [comment]
Since this issue has been fixed in the 4.0.0 by including lang3 package in the plugin, here is a generated locale.txt file you can use instead.
https://www.dropbox.com/s/0o1gg2wansmyijw/locale.txt?dl=0
